src/actions/clusterActions.js

```diff
@@ -186,7 +186,7 @@ export function clusterLoadDetails(
 
       return cluster;
     } catch (error) {
-      if (error.status === StatusCodes.NotFound) {
+      if (error.response.status === StatusCodes.NotFound) {
```

Comment:
I don’t feel safe replacing this altogether. What do you think about having both conditions, but with a logical OR?

Reply:
Something like 
```suggestion
      if (error.status === StatusCodes.NotFound || error.response.status === StatusCodes.NotFound) {
```

Reply:
Please check this out: https://github.com/visionmedia/superagent/issues/1139#issuecomment-267698065

Reply:
It's pretty old, but IIRC, with `axios` is the same thing, you check `error.response`

Reply:
My only fear is that this will no longer work once we switch to the Cluster API, and we'll use our custom client. 
Our custom response objects don't have the `response` property

Reply:
Legit. 

But then you will have all this noisy checks in these ORs.  I'm ok with everything, you decide :-)
For now, this does the job, if you want we can add these extra checks for the future.